### PR TITLE
Updates and fixes

### PR DESCRIPTION
**Pull request by aait**

### CODE REVIEW

File: includes/abstracts/abstract-wc-payment-gateway-payex.php [hunk 1]
```diff
@@ -165,6 +214,13 @@ public function request( $method, $url, $params = array() ) {
 				}
 			}
 
+			// Message for invalid Msisdn
+			if ( ( strpos( $message, 'The Msisdn is not valid' ) !== false ) ||
```
Comment (asbjornu) [inline includes/abstracts/abstract-wc-payment-gateway-payex.php]: Can't you use something else than `message` for this? Isn't `type` or `problems` detailed enough to extract this sort of information?

Comment (asbjornu) [reply includes/abstracts/abstract-wc-payment-gateway-payex.php]: How does the entire `application/problem+json` look like for this validation error? Can't you deserialize the `$message` into an object and look at its `type` or `problems` properties to figure out this in a more programmatic-friendly way than searching through a string after a human readable error message?

File: includes/abstracts/abstract-wc-payment-gateway-payex.php [hunk 2]
```diff
@@ -124,6 +124,57 @@ protected function log( $message, $level = 'notice' ) {
...
+		} catch ( \PayEx\Api\Exception $e ) {
+			if ( $this->debug === 'yes' ) {
+				$time = microtime( true ) - $start;
+				$this->log( sprintf( '[%.4F] Exception: %s', $time, $e->getMessage() ) );
+			}
+
+			$this->response_body = $client->getLastResponse();
+
+			// Message for invalid Msisdn
+			if ( ( strpos( $this->response_body, 'The Msisdn is not valid' ) !== false ) ||
```
Comment (asbjornu) [inline includes/abstracts/abstract-wc-payment-gateway-payex.php]: Please don't search through `response_body` after human readable error messages. Use the `type` or `problems` properties of the `application/problem+json` response.

**Comment by asbjornu** [comment]
My review comments are still not addressed, @aait.

File: swedbank-pay-woocommerce-payments.php [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+ * Plugin Name: Swedbank Pay WooCommerce Payments
+ * Plugin URI: https://www.swedbankpay.se/
+ * Description: Provides a Credit Card Payment Gateway through Swedbank Pay for WooCommerce.
+ * Author: Swedbank Pay
+ * Author URI: https://www.swedbankpay.se/
  * License: Apache License 2.0
  * License URI: http://www.apache.org/licenses/LICENSE-2.0
  * Version: 1.3.1
- * Text Domain: payex-woocommerce-payments
+ * Text Domain: sb-woocommerce-payments
```
Comment (asbjornu) [inline swedbank-pay-woocommerce-payments.php]: Why the abbreviation of `swedbank-pay` to `sb`?

```suggestion
 * Text Domain: swedbank-pay-woocommerce-payments
```

File: swedbank-pay-woocommerce-payments.php [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
- * Plugin Name: PayEx WooCommerce Payments
- * Plugin URI: http://payex.com/
- * Description: Provides a Credit Card Payment Gateway through PayEx for WooCommerce.
- * Author: PayEx
- * Author URI: http://payex.com/
+ * Plugin Name: Swedbank Pay WooCommerce Payments
+ * Plugin URI: https://www.swedbankpay.se/
+ * Description: Provides a Credit Card Payment Gateway through Swedbank Pay for WooCommerce.
+ * Author: Swedbank Pay
+ * Author URI: https://www.swedbankpay.se/
```
Comment (asbjornu) [inline swedbank-pay-woocommerce-payments.php]:
```suggestion
 * Author URI: https://www.swedbankpay.com/
```

**Comment by asbjornu** [comment]
Can you update `README.md` and other references to "PayEx" with "Swedbank Pay" as well?

File: includes/abstracts/abstract-wc-payment-gateway-payex.php (hunk 1 above)
Comment (asbjornu) [reply includes/abstracts/abstract-wc-payment-gateway-payex.php]: This is still unresolved. Treat `problem+json` responses as structured JSON objects, not as a string.

**Comment by aait** [comment]
> There is still one unresolved comment and I wonder if not much of what the module does should be done by `swedbank-pay-sdk-php` instead?

May be. Regarding that message. Method module doesn't use `request1()` method. That's just backup.

File: readme.txt [hunk 5]
```diff
@@ -1,17 +1,17 @@
-=== PayEx WooCommerce Payments ===
+=== Swedbank Pay WooCommerce Payments ===
```
Comment (asbjornu) [inline readme.txt]: `readme.txt` is just repetition of `readme.md` and can be deleted.

**Comment by asbjornu** [comment]
Just delete `readme.txt` and resolve the conflicts in `readme.md` and this should be good to merge.

File: readme.txt (hunk 5 above)
Comment (aait) [reply readme.txt]: Wordpress Plugin catalog requires readme.txt

**Pull request merged**